### ISSUE TITLE
Context menu (Shift+F10) and edit value (Alt+V) via keyboard shortcuts

### ISSUE THREAD

**Issue by mkruselj** [surge-synthesizer/surge#6233]
We should be able to open the context menu on any control we are hovering above, or open the main menu if we're hovering about no controls. If we have keyboard focus active (i.e. screenreaders), this falls through and takes precedence over whatever we're hovering above.

Similarly, add Alt+V as "Edit value" keybinding which will open a typein for any parameter that supports text input.
